csharp/test/Apache.Arrow.Tests/Decimal128ArrayTests.cs

```diff
@@ -97,6 +97,22 @@ public void AppendLargeDecimal()
                     Assert.Equal(-large, array.GetValue(1));
                 }
 
+                [Fact]
+                public void AppendMaxAndMinDecimal()
+                {
+                    // Assert
+                    var builder = new Decimal128Array.Builder(new Decimal128Type(29, 0));
+                    
+                    // Act
+                    builder.Append(Decimal.MaxValue);
```

Comment:
I don't think it would hurt to also test a value slightly less than max, and slightly more than min as well.

```suggestion
                    builder.Append(Decimal.MaxValue);
                    builder.Append(Decimal.MaxValue - 10);
                    builder.Append(Decimal.MinValue + 10);
```

Reply:
done